inference-engine/include/ie_blob.h

```diff
@@ -243,17 +243,6 @@ class INFERENCE_ENGINE_API_CLASS(Blob) {
      * @return The allocator for allocator-based blobs or nullptr if there is none
      */
     virtual const std::shared_ptr<IAllocator>& getAllocator() const noexcept = 0;
-
-    /**
-     * @brief Gets a handle to allocated memory
-     *
-     * @return The handle to allocated memory for allocator-based blobs or nullptr if there is none
-     */
-    virtual void* getHandle() const noexcept = 0;
```

Comment:
It is not backward compatible change and we didn't deprecate this method. What if some customers are using this method from `Blob::Ptr`?

Reply:
It is protected method